examples/svm/plot_svm_scale_c.py

```diff
@@ -67,26 +60,49 @@
 )
 
 # %%
-# Now, we can define a linear SVC with the `l1` penalty.
+# L1-penalty case
+# ---------------
+# In the L1 case, theory says that prediction consistency (i.e. that under
```

Comment:
While updating this example, have you came across a reference to the "theory says ..." claim?

(It would help resolve https://github.com/scikit-learn/scikit-learn/issues/4657)

Reply:
maybe Theorem 5.1. in https://arxiv.org/pdf/0801.1095.pdf ?

Reply:
> maybe Theorem 5.1. in [arxiv.org/pdf/0801.1095.pdf](https://arxiv.org/pdf/0801.1095.pdf) ?

That theorem establishes an approximate equivalence between the Lasso and the Dantzig selector. I don't think that this theorem is related to the claim that "it is not possible for the learned estimator to predict as well as model knowing the true distribution because of the bias of the L1" simply because it is not true that the L1 norm always introduces bias. I really think we should remove such claim.